numpyro/infer/autoguide.py

```diff
@@ -455,12 +456,25 @@ def __call__(self, *args, **kwargs):
 
         return result
 
-    def sample_posterior(self, rng_key, params, sample_shape=()):
+    def sample_posterior(self, rng_key, params, sample_shape=(), *args, **kwargs):
```

Comment:
nit: `*args, sample_shape=(), **kwargs`

Reply:
Thank you for the review! Just corrected the order arguments